src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleBadWifi.kt

```diff
@@ -216,21 +211,11 @@ object ModuleBadWifi : Module("BadWIFI", Category.COMBAT) {
         }
     }
 
-    private fun shouldLag(): Boolean {
-        return 0 < world.countEnemies(0f..8f) && (!player.isUsingItem || !stopWhileUsingItem)
-    }
-
-    @JvmStatic
-    internal fun makeLines(
-        positions: LinkedHashSet<PositionData>, tickDelta: Float
-    ): Array<Vec3> {
-        val mutableList = mutableListOf<Vec3>()
-        positions.forEach {
-            mutableList.add(Vec3(it.vec))
-        }
-        mutableList += player.interpolateCurrentPosition(tickDelta)
-        return mutableList.toTypedArray()
-    }
+    private fun shouldLag() = world.findEnemy(0f..rangeToStart) != null &&
+        (!player.isUsingItem || !stopWhileUsingItem) &&
+        mc.currentScreen == null &&
+        !player.isDead &&
+        !player.isUsingItem
```

Comment:
!player.isUsingItem twice?

Reply:
solutions at 6 am sometimes makes things like this invisible to the eye...